CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 * Refactor CSS away from `color()` function. (UIDATIMP-1577)
 * Match profile Settings: suppress checkboxes and checkbox actions. (UIDATIMP-1496)
 * Display missed options in the 'Existing holdings record field' section after creating match profile. (UIDATIMP-1579)
+* Provide "accountNo" field mapping when account-level accounting code is specified for invoice mapping (UIDATIMP-1584)
```

Comment:
Could you please clarify should we add point for consistency` invoice mapping**.** (UIDATIMP-1584)`